jsonnet/cmd.go

```diff
@@ -203,6 +203,26 @@ func processArgs(givenArgs []string, config *config, vm *jsonnet.VM) (processArg
 		i++
 	}
 
+	handleExtVarVal := func(handle func(key string, val string)) error {
```

Comment:
I would call them handleVarVal/handleVarFile to match the get function names and because they don't just do ExtVar, otherwise LGTM